### PR TITLE
feat(Menu): allow setting submenu position to left

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/80492480/pulses/7110141799

### CODE REVIEW

**Review by YossiSaadi — COMMENTED** [review]
I still think we can have a "position" prop to be able to extend this behavior. It should default to right and have options of right or left.
internally you can do exactly what you do right now, but check if the position is left (the undefined behavior should be position right)
let's discuss it
@talkor wdyt?

File: packages/core/src/components/Menu/MenuItem/__tests__/MenuItem.test.js [hunk 1]
```diff
@@ -24,6 +24,49 @@ describe("<MenuItem />", () => {
     expect(queryAllByText("Sub Item")).toHaveLength(0);
   });
 
+  const submenuPositions = [
+    {
+      submenuPosition: "left",
+      expectedPosition: "left-start"
+    },
+    {
+      submenuPosition: undefined,
```
Comment (YossiSaadi) [inline packages/core/src/components/Menu/MenuItem/__tests__/MenuItem.test.js:33]: is it still like that?

**Review by YossiSaadi — APPROVED** [review]
LGTM!

File: packages/core/src/components/Menu/MenuItem/__tests__/MenuItem.test.js (hunk 1 above)
Comment (rivka-ungar) [reply packages/core/src/components/Menu/MenuItem/__tests__/MenuItem.test.js:33]: yes, it is checking the case where the user doesnt send the prop

**Pull request merged**